app/common/src/map/HotelCard.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+
+import * as React from 'react';
+import { StyleSheet, View, Text, Image } from 'react-native';
+import Color from './../Color';
+
+type Props = {|
+  width: number,
+  hotel: {|
+    name: string,
+  |},
+|};
+
+const styles = StyleSheet.create({
+  description: {
+    flexDirection: 'row',
+  },
+  image: {
+    width: 50,
+    height: 50,
+    borderRadius: 5,
+    marginRight: 10,
+  },
+  hotelName: {
+    fontSize: 18,
+  },
+  metainfo: {
+    color: Color.grey._700,
+    opacity: 0.8,
+  },
+  price: {
+    color: Color.brand,
+    fontWeight: 'bold',
+  },
+});
+
+const HotelCard = ({ width, hotel }: Props) => (
+  <View style={[{ width }]}>
+    <View style={styles.description}>
+      <View>
+        <Image
+          style={styles.image}
+          source={{
+            uri: 'http://q.bstatic.com/images/hotel/square128/117/11753593.jpg',
```

Comment:
Should I care now or we will connect the real data later?

Reply:
I plan to do new PR for real data :)